.framework/node/backend/lib/event.js

```diff
@@ -12,8 +12,12 @@ const axios = axiosLib.create({
 });
 
 async function sendEvent(event, metadata) {
-  const result = await axios.post(`/users/${WILCO_ID}/event`, JSON.stringify({ event, metadata }));
-  return result.data;
+  try {
+    const result = await axios.post(`/users/${WILCO_ID}/event`, JSON.stringify({event, metadata}));
+    return result?.data;
+  } catch (e) {
+    // pass
+  }
```

Comment:
@NivRichter Thoughts about it and this code bother me. I think we should not try-catch here but use a Mock Server instead. Maybe something like [mockoon](https://github.com/mockoon/mockoon)?

@shem8 Would be happy to get your input as well

Reply:
Added mock server to handle the events